pkg/templates/templates_test.go

```diff
@@ -315,6 +316,17 @@ func TestResolveTemplate(t *testing.T) {
 			"value: Raleigh",
 			nil,
 		},
+		{
+			"value: $ocm_encrypted:Eud/p3S7TvuP03S9fuNV+w==",
+			Config{
+				EncryptionConfig: EncryptionConfig{
+					AESKey: otherKey, AESKeyFallback: key, DecryptionEnabled: true, InitializationVector: iv,
```

Comment:
This test case shows it used the fallback key to decrypt, right? Should there be another test to ensure that it tries the "main" key, even if a fallback key was given?

Reply:
Good idea! I just added it.